addons/base_import/models/base_import.py

```diff
@@ -497,6 +514,7 @@ def parse_preview(self, options, count=10):
                 'headers_type': header_types or False,
                 'preview': preview,
                 'options': options,
+                'advanced_mode': options.get('advanced') if options.get('keep_matches') else any([len(models.fix_import_export_id_paths(col)) > 1 for col in headers or []]),
```

Comment:
What's the difference between advanced & keep_matches options? Why do both exist?